### ISSUE TITLE
node-http2 is slow, apparently :(

### ISSUE THREAD

**Issue by domenic** [jussi-kalliokoski/concat-vs-multiplex#2]
https://github.com/molnarg/node-http2/issues/98

On the one hand, this is good news for HTTP/2 in theory.

On the other hand, this is bad news for HTTP/2 in real-world io.js.

**Comment by jussi-kalliokoski** [comment]
Yes, and also buggy (crashes on my machine if you try to load the multiplex examples twice). However, I'd be happy to take PRs that test against another H2 implementation to get a better comparison, it should be relatively straightforward to just use the builds created by the code in here in a better H2 server.
